metadata.mk

```diff
@@ -14,7 +14,7 @@ K8S_VERSION=v1.29.7
 COREDNS_VERSION=1.5.2
 ETCD_VERSION=v3.5.6
 HELM_VERSION=v3.11.3
-KINDEST_NODE_VERSION=v1.29.7
+KINDEST_NODE_VERSION=v1.29.2
```

Comment:
What is the reason for this downgrade? The "Felix: Windows FV capz" was [green](https://tigera.semaphoreci.com/workflows/cdc20da8-a99b-42af-958c-13bf75dbcc2b?pipeline_id=c3c9dbd8-4023-408e-b679-59bdd4671530) when we bump it to v1.29.7 in  https://github.com/projectcalico/calico/pull/9133. Does the newer kindest/node break something?

Reply:
@hjiawei I was trying different things and downgraded it to the lastest v1.29 patch claimed to be supported by kind v0.22.0 (https://github.com/kubernetes-sigs/kind/releases/tag/v0.22.0) in case that was a problem (as upgrading kind would be a whole other can of worms)
I'll revert that and hopefully things will work fine (but we should be mindful of kindest/node versions availability and compatibility, as that is the sole reason we split the env vars into K8S_VERSION and KINDEST_NODE_VERSION)

Reply:
@hjiawei I ended up disassociating this version from the one used by the CAPZ cluster creation tools